Sources/SwagGenKit/CodeFormatter.swift

```diff
@@ -80,11 +80,19 @@ public class CodeFormatter {
         context["filename"] = getFilename(schema.name)
         context["type"] = getModelType(schema.name)
 
-        if case .simple = schema.value.type {
-            context["simpleType"] = getSchemaType(name: schema.name, schema: schema.value)
+        switch schema.value.type {
+        case .simple:
+            let schemaType = getSchemaType(name: schema.name, schema: schema.value)
```

Comment:
Yeah, great! Now if you just move this duplicated line out of this switch statement, `let schemaType = getSchemaType(name: schema.name, schema: schema.value)` and change the template to check for `aliasType` instead of `simpleType` we can merge it. Before that though, you may as well add the array type here as well to this PR to make it easier

Reply:
Done 👍